src/NzbDrone.Core/Indexers/Definitions/BinSearch.cs

```diff
@@ -165,65 +198,141 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
-                var publishDate = DateTimeUtil.FromUnknown(row.QuerySelector("td:nth-child(6)").TextContent);
-                var sizeElement = row.QuerySelector("td > span.d");
-                var size = ParseSizeRegex.Match(sizeElement.TextContent);
-                var infoUrl = string.Format("{0}{1}", _settings.BaseUrl.TrimEnd('/'), row.QuerySelector("a").GetAttribute("href"));
-                var downloadUrl = string.Format("{0}/?action=nzb&{1}=1", _settings.BaseUrl.TrimEnd('/'), guid);
+                var releaseGuid = row.QuerySelector("input[type=checkbox]")?.GetAttribute("name");
+                var releaseUrl = string.Format("{0}/?action=nzb&{1}=1", _settings.BaseUrl.TrimEnd('/'), releaseGuid);
+                var releaseInfoUrl = string.Format("{0}{1}", _settings.BaseUrl.TrimEnd('/'), rowDescription.QuerySelector("a").GetAttribute("href"));
+
+                var trimmedDate = row.LastChild.TextContent.Trim();
+                var releaseDate = DateTime.ParseExact(trimmedDate, new string[] { "dd-MMM-yyyy", "dd-MMM" }, CultureInfo.InvariantCulture);
 
                 var release = new ReleaseInfo
                 {
-                    Guid = guid,
-                    Title = parsedTitle.Groups["title"].Value,
-                    Size = ReleaseInfo.GetBytes(string.Format("{0} {1}", size.Groups["size"].Value, size.Groups["unit"].Value)),
-                    PublishDate = publishDate,
-                    Categories = new List<IndexerCategory> { NewznabStandardCategory.Other },
-                    InfoUrl = infoUrl,
-                    DownloadUrl = downloadUrl
+                    Guid = releaseGuid,
+                    Title = releaseTitle,
+                    DownloadUrl = releaseUrl,
+                    InfoUrl = releaseInfoUrl,
+                    PublishDate = releaseDate
                 };
 
+                var matchReleaseSize = FileSizeRegex.Match(rowDescription.TextContent);
+                if (matchReleaseSize.Success)
+                {
+                    release.Size = ReleaseInfo.GetBytes(string.Format("{0} {1}", matchReleaseSize.Groups["size"].Value, matchReleaseSize.Groups["unit"].Value));
+                }
+
+                var fileTypes = rowDescription.ChildNodes
+                    .Where(node => node.NodeType == NodeType.Text)
+                    .Select(textNode => FileTypesRegex.Match(textNode.TextContent))
+                    .Where(textMatch => textMatch.Success)
+                    .Select(fileTypeMatch => fileTypeMatch.Groups["filetype"].Value);
+
+                if (fileTypes.Count() > 0)
+                {
+                    // Guess category based on filetype
+                    if (fileTypes.Any(fileType => CommonVideoFileTypes.Contains(fileType)))
+                    {
+                        release.Categories = new List<IndexerCategory> { NewznabStandardCategory.Movies, NewznabStandardCategory.TV, NewznabStandardCategory.XXX };
```

Comment:
This is problematic, we don't want to tag everything with XXX

Reply:
Would Movies and TV category be okay though?
Otherwise it is probably better to wait for a uniform implementation (https://github.com/Prowlarr/Prowlarr/issues/394).

Reply:
That's probably fine for now.